package.json

```diff
@@ -39,14 +39,13 @@
     "test": "mocha && npm run lint",
     "coverage": "nyc -x test -x dist/sourcemap-register.js node_modules/mocha/bin/mocha",
     "upgrade": "npm-check -u && npm audit fix",
-    "bump": "bump --tag --push --all && git tag -afm v1 v1 && git push --tags --force",
```

Comment:
Why was this script removed? It looks like the `"release"` script below relies on it

Reply:
It's been so long since I looked at this I can't remember. Maybe an issue I hit when trying to develop on a fork if I had to guess.

Reply:
I think these scripts were calling out to https://github.com/JS-DevTools/version-bump-prompt. I think it's fine to remove the scripts (and the dev dep below on `@jsdevtools/version-bump-prompt`) because:

- It's redundant with the vanilla `npm version` command
- CI will run tests before allowing a release to happen
- Any package under @jsdevtools is relatively unlikely to see any maintenance